src/gql/generated/types.ts

```diff
@@ -111,6 +116,7 @@ export type Mutation = {
   restartTask: Task;
   saveSubscription: Scalars['Boolean'];
   removePatchFromCommitQueue?: Maybe<Scalars['String']>;
+  restartPatch?: Maybe<Scalars['String']>;
```

Comment:
Not sure if this was intentional but looks like it generated some types from a different schema change (but I'm guessing that it'll need to be added anyway)

Reply:
Yeah i guess `graphql-codegen` generates the types for everything in the `schema.graphql` file we pass in. Regardless of if we use them or not